pkgs/servers/ldap/lldap/default.nix

```diff
@@ -1,68 +1,115 @@
 { fetchFromGitHub
 , fetchzip
+, fetchpatch
+, stdenv
 , lib
 , lldap
 , nixosTests
 , rustPlatform
+, wasm-pack
+, wasm-bindgen-cli
+, binaryen
+, which
 }:
```

Comment:
Would be cool if you could keep it alphabetical :)

Reply:
not a fan, but at least it's an objective rule, so ... yeah